Makefile

```diff
@@ -74,8 +74,16 @@ help: ## Describe how to use each target
 gen: $(BUF_V1_MODULE_DATA) $(yarn) ## Generate files from proto files
 	@printf "$(ansi_format_dark)" $@ "generating files..."
 	@$(buf) generate
+	@$(MAKE) gen-wkt
 	@printf "$(ansi_format_bright)" $@ "ok"
 
+generated_dir := website/generated
+gen-wkt: $(protoc) $(protoc-gen-doc)
```

Comment:
Just for my curiosity: what does WKT stand for again?

Reply:
Well-known types https://buf.build/beta/wellknowntypes